pkg/cluster/controllers/cache/redis.go

```diff
@@ -47,13 +47,14 @@ type RedisController struct {
 	Scheme             *runtime.Scheme
 	RedisConnect       *RedisConnect
 	ResourceManager    ResourceManager
-	expectCR, actualCR *redisOp.RedisFailover
+	expectCR, actualCR runtime.Object
```

Comment:
Why use `runtime.Object` instead of `redisOp.RedisFailover`?

Reply:
If use redisOp.RedisFailover, when controller gets actual cr from crd client, the type of crd client return if unstructed.Unstructed, which implements runtime.Object, so we can directly use rc.actualCR=${crdReturn}.

Reply:
👌